### ISSUE TITLE
Allow `biome-ignore` in `eslint(capitalized-comments)`

### ISSUE THREAD

**Issue by hamirmahal** [oxc-project/oxc#21322]

**Comment by hamirmahal** [comment]
It can be cumbersome to have to add

```tsx
// oxlint-disable-next-line capitalized-comments
```

every time we want to ignore a Biome rule.

**Comment by hamirmahal** [comment]
```tsx
// oxlint-disable-next-line capitalized-comments
// biome-ignore lint/style/useComponentExportOnlyModules: We have to export this.
export async function generateMetadata(): Promise<Metadata> {
```

**Comment by bradzacher** [comment]
Question: why are you using biome and oxlint at the same time?

**Comment by hamirmahal** [comment]
Biome does a lot of useful things that Oxlint doesn't, like auto-formatting object keys on save.

**Comment by hamirmahal** [comment]
Maybe there's less of a gap with newer versions, but [versions 1.36+ don't work on my machine](https://github.com/oxc-project/oxc/issues/17510).

**Comment by bradzacher** [comment]
> like auto-formatting object keys on save

What do you mean by "auto formatting"?
And have you seen `oxfmt`...?

Oxlint has the ability to run any eslint plugin in it. And I guarantee there's an ESLint plugin that does what you need.

You shouldn't ever be using both biome and oxlint/oxfmt at the same time.

**Comment by camchenry** [comment]
In the interest of not continuing to expand this exceptions list forever, you can use the configuration of the rule to ignore `biome-ignore` for your case.

For example:
```js
// oxlint.config.ts
{
  rules: {
    "capitalized-comments": [
      "error",
      "always",
      { ignorePattern: "biome-ignore" },
    ],
  },
}
```

```ts
// biome-ignore lint/style/useComponentExportOnlyModules: We have to export this.
export async function generateMetadata(): Promise<Metadata> {}
```
